charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -421,6 +421,14 @@ spec:
         limits:
           cpu: "1"
           memory: 2Gi
+      volumeMounts:
```

Comment:
shouldn't be a condition here to verify if the target serving runtime is hugging face?

Reply:
Hi @spolti , this change only applies to the [kserve-huggingfaceserver](https://github.com/kserve/kserve/blob/8272bde3c1a22e8601e77e9555d4e27a848a773d/charts/kserve-resources/templates/clusterservingruntimes.yaml#L399C9-L399C33) clusterservingruntime, do think it is enough?

Reply:
oh, I didn't expand the file, yes, it is enough :D